src/idl_parser.cpp

```diff
@@ -1335,7 +1335,7 @@ CheckedError Parser::ParseTable(const StructDef &struct_def, std::string *value,
       if (!struct_def.sortbysize ||
           size == SizeOf(field_value.type.base_type)) {
         switch (field_value.type.base_type) {
-          // clang-format off
+// clang-format off
```

Comment:
undo?

Reply:
ughh, this is very annoying bug still in clang-format. 

i ran `scripts/clang-format-git.sh` and it didn't fix it. I then ran `scripts/clang-format-all.sh` and still doesn't fix is, but it does fix a bunch of other issues. That i'm too lazy to exclude from here.

Reply:
i cleaned them up in #6941 and rebased this.